### PR TITLE
Generate static Swagger documentation with Maven (PR2)

### LINKED ISSUE #808: Test generate REST API documentation with Swagger

### PR DESCRIPTION
**Pull request by JAmmermann-DLR**
⚠️Should be reviewed and merged after #833 

- Added documentation generation to the server pom 
- Added generated files
- Minor changes to the model classes

After this PR Swagger Annotations for all Resources and a Documentation Resource can be added (see #834)

Closes #808

### CODE REVIEW

**Comment by SaMuellerDLR** [comment]
@JAmmermann-DLR please merge newest development into this branch so github only shows the actual changes.

**Review by SaMuellerDLR — APPROVED** [review]
Approved!

File: de.dlr.sc.virsat.server/src/de/dlr/sc/virsat/server/resources/ModelAccessResource.java [hunk 1]
```diff
@@ -179,14 +178,14 @@ public Response getCa(@PathParam("caUuid") String caUuid) {
...
-				notes = "<br>This service puts CAs")
+				notes = "This service puts CAs")
 		@ApiResponse(
 				code = HttpStatus.OK_200,
 				message = "Successful operation")
 		@PUT
 		@Path(CA)
 		@Consumes(MediaType.APPLICATION_JSON)
-		public Response putCa(ABeanCategoryAssignment bean) {
+		public Response putCa(@ApiParam(value = "Ca to put", required = true) ABeanCategoryAssignment bean) {
```
Comment (andriikovalov-dlr) [inline de.dlr.sc.virsat.server/src/de/dlr/sc/virsat/server/resources/ModelAccessResource.java:188]:
```suggestion
		public Response putCa(@ApiParam(value = "CA to put", required = true) ABeanCategoryAssignment bean) {
```

**Review by andriikovalov-dlr — APPROVED** [review]
Looks alright to me :)
Just a minor suggestion.

**Pull request merged**